docker-compose.yml

```diff
@@ -17,6 +17,6 @@ services:
       context: backend
       dockerfile: Dockerfile.dev
     ports:
-      - 5000:5000
+      - 8000:8000
```

Comment:
The backend is actually ported through nginx, so we need to update also the nginx configuration.